app_feup/lib/controller/local_storage/app_lectures_database.dart

```diff
@@ -33,7 +33,7 @@ class AppLecturesDatabase extends AppDatabase {
 
     // Convert the List<Map<String, dynamic> into a List<Dog>.
     return List.generate(maps.length, (i) {
-      return Lecture.secConstructor(
+      return Lecture.fromHtml(
```

Comment:
Why was this change introduced?

Reply:
Also, reading from memory without the `ocurrId` might be the cause of the problems

Reply:
Some changes were done manually due to problems in the merging process, so that the code was up-to-date with the develop branch. But I'll try to undo these changes.